tests/test_sklearn_svc.py

```diff
@@ -0,0 +1,118 @@
+"""
```

Comment:
Can we call the file `test_sklearn_svr`? We will add regression sooner or later.

Reply:
you mean `test_sklearn_sv`